modules/templates/helper.go

```diff
@@ -972,3 +974,23 @@ func mirrorRemoteAddress(m models.RemoteMirrorer) remoteAddress {
 
 	return a
 }
+
+// JsPrettyNumber renders a number using english decimal separators, e.g. 1,200 and subsequent
+// JS will replace the number with locale-specific separators, based on the user's selected language
+func JsPrettyNumber(i interface{}) template.HTML {
+	var num int64
+	switch v := i.(type) {
```

Comment:
The type-casting code is the same as `PrettyNumber`, maybe we need a util function to do the same work.

And we can support unsigned types in the unique type-casting function